extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -84,7 +86,11 @@ export class AmpAudio extends AMP.BaseElement {
                   || this.element.getAttribute('aria-label')
                   || doc.title;
     const album = this.element.getAttribute('album');
-    const poster = this.element.getAttribute('poster')
+    if (this.element.getAttribute('poster')) {
```

Comment:
please create an issue and add a TODO here to remove it all together after checking usage.